src/App/App.js

```diff
@@ -346,6 +360,40 @@ function FullApp() {
     setIsSettingsVisible(false);
   };
 
+  const handleEmailEntered = (email) => {
+    // Regular expression for email validation
+    const emailRegex = /^[\w-]+(\.[\w-]+)*@([\w-]+\.)+[a-zA-Z]{2,7}$/;
+
+    if (emailRegex.test(email)) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
`return Boolean(emailRegex.test(email))` for short

Reply:
resolved this in the next PR, thanks!